<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>192</width>
    <height>192</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLauncher">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>btnLauncher</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNewVersionLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>labelNewVersionLog</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>671</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDownloadRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>labelDownloadRate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDownloadStatus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>labelDownloadStatus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateGame">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>btnUpdateGame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>btnTest</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>MenuSetting</string>
    </property>
    <addaction name="actionAutoLaunchGame"/>
    <addaction name="actionAutoUpdateGame"/>
    <addaction name="actionAutoUpdateLauncher"/>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>MenuMisc</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSetting"/>
   <addaction name="menuMisc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAutoLaunchGame">
   <property name="text">
    <string>MenuActionAutoLaunchGame</string>
   </property>
  </action>
  <action name="actionAutoUpdateGame">
   <property name="text">
    <string>MenuActionAutoUpdateGame</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>MenuActionAbout</string>
   </property>
  </action>
  <action name="actionAutoUpdateLauncher">
   <property name="text">
    <string>MenuActionAutoUpdateLauncher</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
